vision-quickstart (1)/app/src/main/java/com/google/mlkit/vision/demo/textdetector/TextRecognitionProcessor.java

```diff
@@ -40,7 +41,8 @@
 public class TextRecognitionProcessor extends VisionProcessorBase<Text> {
 
     private static final String TAG = "TextRecProcessor";
-
+    private ArrayList<String> recogonizedTextList = new ArrayList<String>();
```

Comment:
btw, adding "List" to the end is not required, as long as it is clear there is more than one.  For example, here could be recognizedTexts, the plural is enough to give the information.